services/common/src/components/reports/ReportDetailsForm.tsx

```diff
@@ -98,6 +107,7 @@ const ReportDetailsForm: FC<ReportDetailsFormProps> = ({
   formButtons,
   handleSubmit,
   currentReportDefinition,
+  createPermission = Permission.EDIT_INCIDENTS,
```

Comment:
Couple things:
- I don't think this needs to be a prop, could just be a constant.
- are we sure that the permission to comment on a report is the same permission to edit incidents?
- createPermission sounds like permission to create the report, maybe createCommentPermission?

Reply:
Updated this accordingly.